src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1687,5 +1745,75 @@ static List<MethodInfo> FindMethodsContainingLine(SourceFile sourceFile, int lin
         }
 
         public string ToUrl(SourceLocation location) => location != null ? GetFileById(location.Id).Url.OriginalString : "";
+
+        internal async Task<IEnumerable<AssemblyInfo>> LoadPDBFromSymbolServer(CancellationToken token)
```

Comment:
I think we can simplify these a bit. I would suggest:

- rename this to `ReloadAllPDBsFromSymbolServer`, since that's what it is doing. Unlike the other methods it ignores `asm.TriedToLoadSymbolsOnDemand` and reattempts the load
- Move `internal async Task LoadPDBFromSymbolServer(AssemblyInfo asm, CancellationToken token)` from line 1762 to `class AssemblyInfo`. The method belongs there. You could even pass `SymbolStore` to that method, instead of `DebugStore`
  - and remove the method at 1186
- And rename `LoadSymbolsFromSymbolServer` in `MonoProxy` to `ReloadSymbolsFromSymbolServer`

This avoids the confusion between the different ways the name is getting used, and having slightly different behaviors.

- Example - https://gist.github.com/radical/cf81983de5d8baebfc131694bdd1d6c5